static/js/components/PersonalTab.js

```diff
@@ -57,13 +57,14 @@ class PersonalTab extends ProfileTab {
       )}<br />
       <h4>Where do you live?</h4>
       {this.boundTextField(['city'], 'City')}<br />
-      {this.boundTextField(['state_or_territory'],'State or Territory')}<br />
+      {this.boundStateSelectField(['state_or_territory'], ['country'], 'State or Territory')}<br />
```

Comment:
I agree, but can we postpone this until https://github.com/mitodl/micromasters/issues/217?